client/js/screens/SingleSelfGuided/SingleSelfGuided.js

```diff
@@ -5,152 +5,183 @@ import styles from './styles';
+                        tour: tour,
+                      });
+                    }
+                  }}>
+                  <Text style={styles.playAudio}>
+                    {tour.audio && tour.audio.markers.length > 1 ? 'Play' : ''}
+                  </Text>
+                </TouchableOpacity>
               </View>
-            </View>
-          </View>
-          <Text style={styles.infoText}>Audio</Text>
-          <View style={styles.inputFieldStyles}>
-            <Image
-              style={styles.eventTextFields}
-              source={require('../../assets/images/icFilterAudio.png')}
-            />
-            <Text style={styles.audioInfo}>
-              {tour.audio && tour.audio.markers.length > 1
-                ? `${tour.audio.markers.length} Points`
-                : 'No Audio'}
-            </Text>
-            <TouchableOpacity>
-              <Text style={styles.downloadAudio}>
-                {tour.audio && tour.audio.markers.length > 1 ? 'Download' : ''}
-              </Text>
-            </TouchableOpacity>
-            <TouchableOpacity
-              onPress={() => {
-                if (tour.audio) {
-                  return navigation.push('StoryTrail', {
-                    tour: tour,
-                  });
-                }
-              }}>
-              <Text style={styles.playAudio}>
-                {tour.audio && tour.audio.markers.length > 1 ? 'Play' : ''}
-              </Text>
-            </TouchableOpacity>
-          </View>
 
-          <View>
-            <Text style={styles.eventInfoTitle}>About the Trails:</Text>
+              <View>
+                <Text style={styles.eventInfoTitle}>About the Trails:</Text>
 
-            <Text style={styles.eventInfoDescription}>{tour.description}</Text>
-          </View>
-          <View style={styles.eventMapContainer}>
-            <MapView
-              style={styles.eventMap}
-              mapType="satellite"
-              region={{
-                latitude: 50.922148,
-                longitude: -119.643196,
-                latitudeDelta: 0.02,
-                longitudeDelta: 0.0121,
-              }}>
-              {tour.audio &&
-                tour.audio.markers.map(each => (
-                  <Marker
-                    key={each.id}
-                    coordinate={{
-                      latitude: each.latitude,
-                      longitude: each.longitude,
-                    }}
-                    title={each.title}
-                    description={each.description}
-                  />
-                ))}
-            </MapView>
+                <Text style={styles.eventInfoDescription}>
+                  {tour.description}
+                </Text>
+              </View>
+              <View style={styles.eventMapContainer}>
+                <MapView
+                  style={styles.eventMap}
+                  mapType="satellite"
+                  region={{
+                    latitude: 50.922148,
```

Comment:
Why are you having hard coded values for location here?

Reply:
Because we don't have time to get location (lat and long) for all events and tours, that wa snot given by Client.